dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/deduplication/DeduplicationController.java

```diff
@@ -229,11 +221,11 @@ private void checkDbAndRequestStatus( PotentialDuplicate potentialDuplicate,
                 + DeduplicationStatus.MERGED.name() );
     }
 
-    private void checkDeduplicationStatusRequestParam( String status )
+    private void checkDeduplicationStatusRequestParam( DeduplicationStatus status )
```

Comment:
This shouldn't be necessary then

Reply:
yes this is old code so I need to review with your suggestion